docs/cn/Configuring-Alluxio-with-S3.md

```diff
@@ -32,6 +32,25 @@ priority: 0
 
 更改完成后，Alluxio应该能够将S3作为底层文件系统运行，你可以尝试[使用S3在本地运行Alluxio](#running-alluxio-locally-with-s3)
 
+## 使用EC2实例配置文件和IAM角色访问S3
+
+如果你不设置指明`<AWS_ACCESS_KEY_ID>`和`<AWS_SECRET_ACCESS_KEY>`，那么系统假定你在CE2中以一个IAM角色运行Alluxio，并且该IAM角色拥有给定S3 bucket的完整访问权限。
+```json
```

Comment:
Thanks, Chaomin. Do you mean `<AWS_ACCESS_KEY_ID>` and `<AWS_SECRET_ACCESS_KEY>`? If so, should I also update the English version now?